src/main/resources/webroot/index.html

```diff
@@ -216,7 +125,8 @@ <h2>Create a new Vert.x application</h2>
           <div class="col-sm-8">
             <div class="btn-group">
               <label class="btn btn-sm btn-default" ng-repeat="language in vm.languages"
-                     ng-model="$parent.vm.vertxProject.language" uib-btn-radio="language">{{ language | capitalize }}</label>
+                     ng-model="$parent.vm.vertxProject.language" uib-btn-radio="language">{{ language | capitalize
```

Comment:
Can you revert the whitespaces changes please?

Reply:
👷 Done

This was caused by IntelliJ Idea setting called Hard line something. I've just put new configuration value to .editorconfig max_line_length = 140 this should be enough and nobody should have this problem again